app/lib/service/security_advisories/backend.dart

```diff
@@ -123,6 +130,21 @@ class SecurityAdvisoryBackend {
     });
   }
 
+  String _computeDisplayUrl(List<String> idAndAliases) {
+    final githubId =
+        idAndAliases.firstWhereOrNull((id) => id.startsWith('GHSA'));
+    if (githubId != null) {
+      return 'https://github.com/advisories/$githubId';
+    }
+
+    final cveId = idAndAliases.firstWhereOrNull((id) => id.startsWith('CVE'));
+    if (cveId != null) {
+      return 'https://osv.dev/vulnerability/$cveId';
+    }
+
+    return idAndAliases.first;
```

Comment:
Maybe we should just return null if we cannot find a url.

The client probably has to handle null here anyways.

Reply:
We agreed to show the plain id in this case

Reply:
Yeah - but I think the client should decide on that. 'pub_display_url' suggests that it contains a url

Reply:
Actually since we in pub-dev always fetch advisories from osv, we will be sure to find it at osv.dev.